lib/src/main/kotlin/com/lemonappdev/konsist/core/declaration/KoTypeDeclarationImpl.kt

```diff
@@ -45,6 +45,10 @@ internal class KoTypeDeclarationImpl private constructor(
 
     override val isNullable: Boolean by lazy { ktTypeReference.text.last() == '?' }
 
+    override val isGenericType: Boolean by lazy {
+        sourceType.contains("<") && sourceType.contains(">")
```

Comment:
Please use regex:
```kotlin
val pattern = ".+<.+>.+".toRegex()

println(pattern.matches("List<String>")) // True
println(pattern.matches("<String>")) // False
println(pattern.matches("List<>")) // False
```

Reply:
I changed this regex to ```val regex = "\\w+<[^<>]+>".toRegex()``` because your version uses dot sign and this matches any character, including angle brackets and whitespace which causes the tests to fail.